assets.go

```diff
@@ -320,8 +321,8 @@ varying vec2 var_TexCoords;
 void main() {
   var_Color = in_Color;
   var_TexCoords = in_TexCoords;
-  gl_Position = vec4((4.0*in_Position.x) / uf_Projection.x - 2.0,
-  					 (4.0*in_Position.y) / -uf_Projection.y + 2.0, 0, 2.0);
+  gl_Position = vec4(in_Position.x / uf_Projection.x - 1.0,
+  					 in_Position.y / -uf_Projection.y + 1.0, 0, 1.0);
```

Comment:
Makes more sense this way. `uf_Projection` is most often half of the screen, which is exactly what you'd expect if you were to normalize it. The `+/- 1.0` are for alignment.